packit_service/constants.py

```diff
@@ -56,6 +56,7 @@
     "in the linked issue.\n\n"
 )
 NOTIFICATION_REPO = "https://github.com/packit/notifications"
+PACKIT_APP_PREFIX = "packit-as-a-service"
```

Comment:
would it make sense to instead use [this method](https://github.com/packit/packit-service/blob/50e1c9cd220ab4c3f875b1bda15dd701bd98180e/packit_service/config.py#L268)?

Reply:
It may behave slightly differently, if both stg and prod were enabled in the repo, then both will comment if we use the linked method. But I think that's what we want. I will modify my code.